src/analytics/Properties.tsx

```diff
@@ -776,21 +776,21 @@ interface FiatExchangeEventsProperties {
   [FiatExchangeEvents.cico_landing_how_to_fund]: undefined
   [FiatExchangeEvents.cico_currency_chosen]: {
     flow: FiatExchangeFlow
-    currency: AnalyticsCurrency
+    currency: string
```

Comment:
this is being changed from `AnalyticsCurrency` to string to allow any token symbol to be passed here. In the future, this allows for adding new CICO currencies without wallet changes.